doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -482,7 +483,25 @@ If the configuration is successfully downloaded, the system asks for the followi
 * Security server code (**reference data: 2.4**), which is chosen by the security server administrator and which has to be unique across all the security servers belonging to the same X-Road member.
 * Software token’s PIN (**reference data: 2.5**). The PIN will be used to protect the keys stored in the software token. The PIN must be stored in a secure place, because it will be no longer possible to use or recover the private keys in the token once the PIN has been lost.
 
-### 3.4 Configuring configuration backup encryption
+### 3.4 Configuring firewall
+
+It is strongly recommended to protect the security server from unwanted access using a firewall (hardware or software based). The firewall can be applied to both incoming and outgoing connections depending on the security requirements of the environment where the security server is deployed. 
```

Comment:
"Security Server" should be written using upper case initials - "Security Server", not "security server". However, other parts of the document aren't currently consistent either. It would be better to update the whole document so that "Security Server" is used consistently.

Reply:
Updated the entire file.